src/metrics.js

```diff
@@ -20,7 +20,10 @@ async function recordTransactionMetric (fields, update) {
 }
 
 export async function recordTransactionMetrics (transaction) {
-  if (!transaction.response || typeof transaction.response.timestamp !== 'number') {
+  if (
+      !transaction.response || 
+      typeof transaction.response.timestamp !== 'object' // Date object
```

Comment:
I think an `instanceof` check for `Date` would be better here if we know that it will be a date.